components/root/root.jsx

```diff
@@ -269,6 +269,10 @@ export default class Root extends React.PureComponent {
         }
 
         Utils.applyTheme(this.props.theme);
+
+        if (this.props.firstTimeOnboarding) {
```

Comment:
this makes sure of saving the new onboarding preferences during the mount life-cycle.